WooCommerce/Classes/ViewRelated/Themes/ThemesPreviewView.swift

```diff
@@ -63,21 +64,29 @@ struct ThemesPreviewView: View {
     }
 
     @Environment(\.dismiss) var dismiss
+
+    @ObservedObject private var viewModel: ThemesPreviewViewModel
+
     @State private var selectedDevice: PreviewDevice = PreviewDevice.defaultDevice
+    @State private var showPagesMenu: Bool = false
+
     private let theme: WordPressTheme
 
     /// Triggered when "Start with this theme" button is tapped.
     var onStart: () -> Void
```

Comment:
Not related to this PR: 

"Start with This Theme" button text should be "Use This Theme" when in Theme settings page. 
<image src="https://github.com/woocommerce/woocommerce-ios/assets/524475/f9882073-04ec-48fb-808c-41f6cd856227" width="320"/>

1. The button text needs to be fixed.
2. Using a common name for this callback might be better.

Reply:
I will handle the button name fix while working on this https://github.com/woocommerce/woocommerce-ios/issues/11494 task.

Reply:
name update in https://github.com/woocommerce/woocommerce-ios/pull/11492/commits/5657fb6594f6ba8eef11f0a8aa1e3b9e27ba53f3